Chore/update gradio template

deep-diver opened the pull request:
Closes #42

sayakpaul commented on training_pipeline/apps/gradio/semantic_segmentation/app.py:
```diff
@@ -88,7 +86,7 @@ def get_predictions(image: Image):
     title=title,
     description=description,
-    examples=[["test-image.jpg"]],
+    examples=[["test-image1.png"], ["test-image2.png"]],
```
Cute animals. Can we a few more cute pics of dogs and cats? :D

sayakpaul commented on training_pipeline/apps/gradio/semantic_segmentation/palette.txt:
```diff
@@ -1,35 +1,3 @@
-[255, 84, 127]
-[255, 170, 127]
\ No newline at end of file
+[0, 10, 146]
+[38, 0, 44]
+[255, 232, 0]
```
Short and simple. 

Maybe add a comment on top of this? 

* 0 for background 
* 1 for dog
* 2 for cat

sayakpaul reviewed (approved):
LGTM, just minor comment.

deep-diver commented:
@sayakpaul 

added more example photos from TensorFlow official tutorial, 
added comments oh palette.txt
- actually, this tutorial classifies inner, outer, border of an object
